src/listmode_buildblock/SPECTListModeData.cxx

```diff
@@ -0,0 +1,93 @@
+/*!
+  \file
+  \ingroup listmode
+  \brief Implementation of class stir::SPECTListModeData
+
+  \author Daniel Deidda
+  \author Kris Thielemans
+*/
+/*
+    Copyright (C) 2003, Hammersmith Imanet Ltd
+    Copyright (C) 2014, University College London
+    Copyright (C) 2019, National Physical Laboratory
+    This file is part of STIR.
+
+    This file is free software; you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation; either version 2.1 of the License, or
+    (at your option) any later version.
+
+    This file is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+
+    See STIR/LICENSE.txt for details
+*/
+
+#include "stir/listmode/SPECTListModeData.h"
+#include "stir/ExamInfo.h"
+#include "stir/is_null_ptr.h"
+
+START_NAMESPACE_STIR
+
+SPECTListModeData::
+SPECTListModeData()
+{
+}
```

Comment:
I'd remove this. The default constructor will do the same. Then this file can be deleted as far as I can see.